### PR TITLE
Enhanced visibility of result for Notify ingredient

### PR DESCRIPTION
**Pull request by JensonWee**

### CODE REVIEW

File: src/main/java/seedu/kitchenhelper/notification/IngredientNotification.java [hunk 1]
```diff
@@ -65,8 +65,8 @@ public Date stringToDate(String date1) {
 
     public void addToNotification(ArrayList<Ingredient> ingredientArrayList) {
         for (Ingredient i : ingredientArrayList) {
-            notification += i.getIngredientName() + "|" + i.getQuantity() + "|"
-                    + i.getPrice() + "|" + i.getExpiryDate() + "\n";;
+            notification += "Ingredient name : " + i.getIngredientName() + " | " + i.getQuantity() + " portion(s) | $"
```
Comment (yantingsanity) [inline src/main/java/seedu/kitchenhelper/notification/IngredientNotification.java]: image

As seen from the image above, just a suggestion but perhaps you can put like spaces in between to split the expiring ingredients, low in stock ingredients and expired ingredients? for better user readability?

**Review by yantingsanity — COMMENTED** [review]
other than the readability, LGTM!

File: src/main/java/seedu/kitchenhelper/notification/IngredientNotification.java (hunk 1 above)
Comment (zi-hui) [reply src/main/java/seedu/kitchenhelper/notification/IngredientNotification.java]: Good job on the changes made but I believe there are some typos made, should be "These ingredients are currently low in stock" and "These ingredients have expired"

Comment (JensonWee) [reply src/main/java/seedu/kitchenhelper/notification/IngredientNotification.java]: Updated based on suggestions.

**Pull request merged**